### DISCUSSION TITLE
Make the Orange mini-map color more red to distinguish from yellow

### DISCUSSION THREAD

**Discussion by zenseii** [ihhub/fheroes2#7581] [General]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

Currently we aren't using the original hues for some of the colors because they were indistinguishable from certain terrains - e.g. orange with wasteland. But, I find orange and yellow to be too similar. Maybe we could make the orange color a bit more red? We will have to take into account that the color cannot be the same as any terrain on the mini-map.

Related PR with explanation for color changes #6022

Example: Yellow castle upper left, orange castle upper right:

![image](https://github.com/ihhub/fheroes2/assets/12501091/e2a2d601-f1d5-400a-965e-507cb4ab3693)

### Describe the possible solution

Change the color of orange on the minimap so that it is easier to distinguish.

**Comment by zenseii** [comment]
I'm changing this to a discussion because not everyone might agree.

**Comment by XCOM-HUB** [comment]
I agree with making the Orange and Yellow Hero colors more distinct from one another, and also with darkening the Purple Hero color so it’s easier to spot on the mini‑map.

<img width="1281" height="996" alt="Mini-Map Hero Colors" src="https://github.com/user-attachments/assets/9be1a207-c6cb-4ed7-bd0e-1a4f76cf1759" />
